Kubernetes/omsagent.yaml

```diff
@@ -18,6 +18,7 @@ rules:
         "nodes/stats",
         "nodes/metrics",
         "nodes/spec",
+        "nodes/proxy",
```

Comment:
do we need this ?

Reply:
yes, for /pods call.

Reply:
dont we have access to nodes already (meaning nodes/*) ?

Reply:
there is no /nodes/* route and /pods requires "nodes/proxy" resource access.